src/styles.scss

```diff
@@ -216,3 +231,39 @@ body {
   flex-direction: column;
   overflow-x: hidden !important;
 }
+
+.markdown-description {
+  @apply prose whitespace-normal max-w-none;
+  @apply prose-img:rounded-lg prose-img:my-3 prose-img:object-cover prose-img:w-full;
+  @apply prose-a:text-[--link-color];
+  @apply prose-blockquote:mx-0;
+  @apply prose-code:rounded-lg;
+  @apply prose-table:border-collapse;
+  @apply prose-th:px-[13px] prose-th:py-[6px] prose-th:border prose-th:border-[--tw-prose-td-borders] prose-th:align-middle prose-th:border-solid;
+  @apply prose-td:px-[13px] prose-td:py-[6px] prose-td:border prose-td:border-[--tw-prose-td-borders] prose-td:align-middle prose-td:border-solid;
+
+  // table {
+  //   display: block;
+  //   max-width: fit-content;
+  //   overflow-x: auto;
+  //   text-indent: 0;
+  //   border-color: inherit;
+  //   border-collapse: collapse;
+  // }
+
+  pre code {
+    @apply p-0;
+    border: initial;
+    line-height: 1.5;
+  }
+}
+
+#showMoreOverlay {
```

Comment:
Should we really use an ID?

Also, maybe we should use kebap case over camel case?